### PR TITLE
Review Assignments | Back link is broken

### LINKED ISSUE #6862: Review Assignments | Back link is broken
```
      <Button
        name="go-back"
        button="primary"
        to="/schedule/build/upload">
        Go back
      </Button>
```

That object should be styled like a button, but needs to have the functionality of a link. Right now, the button object doesn't have a 'to' property, so clicking on the button doesn't do anything.

### PR DESCRIPTION
**Pull request by nolawi**
Resolves #6862 

### Description
Updates button to use the link
Removes hover color in dropdown button component (unrelated)

Unrelated issue, secondary button in FE toolkit is passing old outline class, created an issue to update. 
https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit/issues/41, for now its primary

screen shot 2018-09-07 at 2 38 17 pm

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "client/node_modules"
   ],
   "dependencies": {
-    "@department-of-veterans-affairs/caseflow-frontend-toolkit": "https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit#95d11a0",
+    "@department-of-veterans-affairs/caseflow-frontend-toolkit": "https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit#issue31",
```
Comment (nolawi) [inline client/package.json]: temp pointing to the fe branch for testing.. will update before merge

Comment (sharonwarner) [reply client/package.json]: Sounds good!

**Comment by shanear** [comment]
stripping ready to merge since this bad boy has been failing for 4 days

**Comment by nolawi** [comment]
looks like the issue is fixed. merging  @sharonwarner

**Pull request merged**